### PR TITLE
removed p tags not needed.

### PR DESCRIPTION
**Pull request by Hallidayo**
## What kind of change does this PR introduce?

Docs update, related to issue #6331 

## What is the current behavior?

Noticed that on #6334 that the contrast on the `p` tag was not being picked up.

## What is the new behavior?

Added a `span` tag to the `p` tag and the contrast is now displaying correctly

Light Mode

Screenshot 2022-04-12 at 14 18 51

Dark Mode

Screenshot 2022-04-12 at 14 19 04

### CODE REVIEW

File: www/pages/auth/Auth.tsx [hunk 1]
```diff
@@ -132,9 +132,11 @@ function AuthPage() {
                 </div>
               </p>
               <h4 className="h4">All the social providers</h4>
-              <p className="p text-base">
-                Enable social logins with the click of a button. Google, Facebook, GitHub, Azure,
-                Gitlab, Twitter, Discord, and many more.
+              <p>
+                <span className="p text-base">
```
[the remark this answers is not in the document]
Comment (Hallidayo) [reply www/pages/auth/Auth.tsx:136]: I found that for some reason at the moment the classes on the p tag are being ignored in the browser on the live site and on local:

Screenshot 2022-04-12 at 17 12 38

[the remark this answers is not in the document]
Comment (MildTomato) [reply www/pages/auth/Auth.tsx:136]: @Hallidayo Can you screenshot the code?

[the remark this answers is not in the document]
Comment (Hallidayo) [reply www/pages/auth/Auth.tsx:136]: Yeah sure, this is the master: 

Screenshot 2022-04-14 at 08 58 45

[the remark this answers is not in the document]
Comment (MildTomato) [reply www/pages/auth/Auth.tsx:136]: ok, i'm just investigating this - seems like another issue

[the remark this answers is not in the document]
Comment (MildTomato) [reply www/pages/auth/Auth.tsx:136]: i pushed some changes, the icons above were in a `<p>` tag which seemed to be breaking it. 🤷‍♂️

[the remark this answers is not in the document]
Comment (Hallidayo) [reply www/pages/auth/Auth.tsx]: Ah right that’s great Thankyou.

**Comment by Isaiah-Hamilton** [comment]
Closing because of updated and merged pull request https://github.com/supabase/supabase/pull/6661

**Pull request closed without merging**